tests/Dummy/GoldenKeyAuthMiddleware.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace BrandEmbassyTest\Slim\Dummy;
+
+use BrandEmbassy\Slim\Middleware;
+use BrandEmbassy\Slim\Request\RequestInterface;
+use BrandEmbassy\Slim\Response\ResponseInterface;
+use Exception;
+
+final class GoldenKeyAuthMiddleware implements Middleware
+{
```

Comment:
chceme tady toto tady mit?

Reply:
Je to dummy testovací kod použitý v testu